ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterManager.java

```diff
@@ -427,7 +445,7 @@ void waitForInitialization() throws InterruptedException {
   /**
    * A callback class used to query information from the {@link HelixClusterManager}
    */
-  class HelixClusterManagerCallback {
+  public class HelixClusterManagerCallback {
```

Comment:
Does not. Changed. (It is an inner class of a package private class, so it was not really public).